README.md

```diff
@@ -408,6 +408,15 @@ render((
         uiSchema={uiSchema} />
 ), document.getElementById("app"));
 ```
+
+For fields for which the order is OK already or a fixed order is not important, you can insert a wildcard `"*"` item in your `ui:oder` definition. All unreferenced fields will be rendered at that point:
```

Comment:
`ui:oder` should be `ui:order`. Also, maybe we should add a note here saying that the other fields will be rendered in the default order, but I see that we don't actually say what the default order is..

You later add an example where `*` shows up in the middle of fields. Is it maybe a good idea to emphasize that "unreferenced" fields means "unreferenced throughout the list" and not just "unreferenced up to that point"?